app/views/links/_form.html.erb

```diff
@@ -12,8 +12,13 @@
   <% end %>
 
   <form class="form-horizontal">
+    <div class="form-group" data-toggle="buttons">
+      <label class="btn btn-private pull-right">
+        <%= check_box_tag :private_link %> This link is
```

Comment:
Og så skal du også fjerne dit form tag, for du har nu en form inde i en anden form (se `form_for` øverst i filen), og det kan man ikke i HTML.